app/views/people/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<ul>
+  <% @all_people.each do |person| %>
+    <li>
+      <%= link_to "#{person[:name]}", "/people/#{person[:id]}" %>
+      --
+      <%= link_to "#{person.tasks.count} Games", "/people/#{person[:id]}/tasks" %>
+      Recommended
+      <!-- -- <%= Task.where(person[:id]).where(completed_at: nil).count %> Incomplete -->
```

Comment:
It could be this line causing issues on this page since using the HTML comment syntax for the Ruby code in the views doesn't always work the way you expect